src/Nokia_LCD.h

```diff
@@ -202,6 +202,6 @@ class Nokia_LCD {
 
     const uint8_t kClk_pin, kDin_pin, kDc_pin, kCe_pin, kRst_pin, kBl_pin;
     uint8_t mX_cursor, mY_cursor;
-    bool inverted = false;
-    bool usingBacklight;
+    bool kInverted = false;
```

Comment:
Well, since you are changing this one as well, then it should be `mInverted` not `kInverted` (the `k` prefix represents constants while the `m` member variables).